services/policies/pkg/config/policies/files.rego

```diff
@@ -0,0 +1,20 @@
+package files
+
+import future.keywords.if
+import data.utils
+
+default granted = true
+
+ALLOWED_FILE_EXTENSIONS := [
```

Comment:
Q: why not making the allowed extensions configurable via an envvar and defaulting them to these values?